lib/model/model.go

```diff
@@ -1968,7 +1969,10 @@ func (s *indexSender) sendIndexTo(ctx context.Context) error {
 		if f.IsReceiveOnlyChanged() {
 			f.Version = protocol.Vector{}
 		}
-		f.LocalFlags = 0 // never sent externally
+
+		// never sent externally
+		f.LocalFlags = 0
+		f.VersionHash = nil
```

Comment:
Again, in theory it does not hurt to send this.

Reply:
Which remind me that we need to specify and document the blocks hash (which is one thing that makes it hurt to send it ... ;)

Reply:
There's also the thing about trusting this from the outside. It's important that it's consistent, so we will generally want to just compute it ourselves if/when we need it.

Reply:
Yeah, I need to document the new config flags too.

Reply:
I fixed disablefsync for you at least :)

Reply:
Anyway, it's easy to start sending this if we change our minds. Trickier to stop sending it once someone depends on it...